Earthfile

```diff
@@ -343,6 +343,15 @@ packaging-full-test:
     BUILD +deb-runtime-test
     BUILD +rpm-runtime-test
 
+check-format:
+    FROM python:3.11.2-slim-buster
+    RUN pip install pipx
+    COPY etc/clang-format*.sh /X/etc/
+    COPY .evergreen/init.sh /X/.evergreen/
+    RUN bash /X/etc/clang-format.sh  # Does nothing, but warms the cache
+    COPY --dir .clang-format src /X/
+    RUN bash /X/etc/clang-format-all.sh --dry-run -Werror
```

Comment:
Can `bash` be omitted when invoking the script(s) to encourage using/respecting shebangs within the file(s)?

Reply:
Yes, but also it can help the error messages be more clear if Bash isn't available (Otherwise you get an ENOENT when trying to execute the file, which can be unintuitive) and doesn't require the file to be executable.